### PR TITLE
added PANOS 9 details

### LINKED ISSUE #930: Update Palo Alto documentation for 9.x
Support for Palo Alto 9.x [was added in Graylog 3.2.2](https://www.graylog.org/post/announcing-graylog-v3-3-2). 

It appears that the [existing Palo Alto section](https://docs.graylog.org/en/3.3/pages/integrations/inputs/palo_alto_networks_input.html) in the Graylog documentation may need to be updated to reflect the new 9.x version support.

### PR DESCRIPTION
**Pull request by jalogisch**
I have added the information about Pan OS 9 input

FIX: #930

### CODE REVIEW

File: pages/integrations/inputs/palo_alto_networks_input.rst [hunk 1]
```diff
@@ -82,4 +85,26 @@ Version 7.1
...
+
+* `9.1 - Traffic Log Fields <https://docs.paloaltonetworks.com/pan-os/9-1/pan-os-admin/monitoring/use-syslog-for-monitoring/syslog-field-descriptions/traffic-log-fields.html>`_
+* `9.1 - Threat Log Fields <https://docs.paloaltonetworks.com/pan-os/9-1/pan-os-admin/monitoring/use-syslog-for-monitoring/syslog-field-descriptions/threat-log-fields.html>`_
+* `9.1 - System Log Fields <https://docs.paloaltonetworks.com/pan-os/9-1/pan-os-admin/monitoring/use-syslog-for-monitoring/syslog-field-descriptions/system-log-fields.html>`_
+
+Version 9.0
+
+* `9.0 - Traffic Log Fields <https://docs.paloaltonetworks.com/pan-os/9-0/pan-os-admin/monitoring/use-syslog-for-monitoring/syslog-field-descriptions/traffic-log-fields.html>`_
+* `9.0 - Threat Log Fields <https://docs.paloaltonetworks.com/pan-os/9-0/pan-os-admin/monitoring/use-syslog-for-monitoring/syslog-field-descriptions/threat-log-fields.html>`_
+* `9.0 - System Log Fields <https://docs.paloaltonetworks.com/pan-os/9-0/pan-os-admin/monitoring/use-syslog-for-monitoring/syslog-field-descriptions/system-log-fields.html>`_
```
Comment (waab76) [inline pages/integrations/inputs/palo_alto_networks_input.rst]: The PAN-OS 9.x Input supports the following log types: CONFIG, CORRELATION, HIP, GLOBAL_PROTECT (the only one that differs significantly between 9.1 and 9.1.3), SYSTEM, THREAT, and TRAFFIC

**Review by waab76 — DISMISSED** [review]
May want to include the full list of supported log types, but otherwise looks good to me.

File: pages/integrations/inputs/palo_alto_networks_input.rst (hunk 1 above)
Comment (jalogisch) [reply pages/integrations/inputs/palo_alto_networks_input.rst]: changed the lists to in the PR

**Review by asachs01 — APPROVED** [review]
LGTM 👍

**Review by waab76 — APPROVED** [review]

**Pull request merged**